integration/coder-app-open-in/main.tf

```diff
@@ -17,13 +17,6 @@ resource "coder_agent" "dev" {
   dir  = "/workspace"
 }
 
-resource "coder_app" "window" {
-  agent_id = coder_agent.dev.id
-  slug     = "window"
-  share    = "owner"
-  open_in  = "window"
-}
```

Comment:
Should we change it to `tab` instead of removing it from the integration testing altogether?